samples/snippets/src/main/java/com/example/spanner/BatchWriteAtLeastOnceSample.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_batch_write_at_least_once]
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.MutationGroup;
+import com.google.cloud.spanner.Options;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.common.collect.ImmutableList;
+import com.google.rpc.Code;
+import com.google.spanner.v1.BatchWriteResponse;
+
+public class BatchWriteAtLeastOnceSample {
+
+  /***
+   * Assume DDL for the underlying database:
+   * <p>
```

Comment:
Code samples in Javadoc should use a `<pre>{@code ... }` construct. See https://github.com/googleapis/java-spanner/blob/59cec9b9cdad169bd8de8ab7b264b04150dda7fb/google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClient.java#L61 for an example